build.sbt

```diff
@@ -34,11 +34,9 @@ val noPublish = List(
 )
 
 // === CI/CD settings ===
-val scala213 = "2.13.11"
-val scala3   = "3.3.0"
+val scala3   = "3.4.2"
```

Comment:
It's recommended for libraries to stay on the LTS version (3.3) if they don't need specific features from the non-LTS version.